src/CustomMySQL.js

```diff
@@ -115,7 +116,19 @@ export default class CustomMySQL {
     }
 
     build () {
-        this._cache = Array.from(arguments);
+
+        const args = Array.from(arguments);
+
+        if (args.length === 1 && squel.cls.isSquelBuilder(args[0])) {
```

Comment:
```js
        this._cache = Array.from(arguments);

        if (squel.cls.isSquelBuilder(this._cache[0])) {

            const query = this._cache[0].toParam();

            this._cache[0] = query.text;
            this._cache.splice(1, 0, query.values);
        }
```
implemented like this to avoid the arguments length checking.